src/main/java/mate/controller/cars/RemoveDriverFromCarController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller.cars;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/remove")
```

Comment:
/delete
Be consistent with namings

Reply:
I think its better to make a difference when are soft deleting object and when we just unlink driver from car